index.bs

```diff
@@ -128,8 +128,11 @@ therefore not shareable, a WebSocket connection is very close to identical to an
 To <dfn id=concept-websocket-establish>establish a WebSocket connection</dfn>, given a
 |url|, |protocols|, and |client|, run these steps:
 
-1. Let |requestURL| be a copy of |url|, with its <a for=url>scheme</a> set to "`http`", if |url|'s
- <a for=url>scheme</a> is "`ws`", and to "`https`" otherwise.
+1. Let |requestURL| be a copy of |url|.
+1. If |requestURL|'s <a for=url>scheme</a> is "`ws`", set |requestURL|'s <a for=url>scheme</a>
+   to "`http`".
+1. Otherwise, if |requestURL|'s <a for=url>scheme</a> is "`wss`", set |requestURL|'s
+   <a for=url>scheme</a> to "`https`".
```

Comment:
Let's do this normalization upfront in the constructor so it doesn't leak out in a number of places.